src/libs/LoginUtils.js

```diff
@@ -1,5 +1,15 @@
 import CONST from '../CONST';
 
+/**
+ * Remove the MERGED_0@ prefix from merged account emails.
+ *
+ * @param {String} email
+ * @returns {String}
+ */
+function getEmailWithoutMergedAccountPrefix(email) {
+    return email.replace(/^MERGED_0@/, '');
```

Comment:
NAB, maybe `MERGED_0@` can be a `CONST`?

Reply:
Question: Is the intention to call this against any string or a single login? Should we expect multiple occurrences? I _think_ this might only replace the first we find.

Reply:
Yes, this will only replace the first occurrence, but that was my intention. It takes an email, and for merged accounts the email we get from the API is like `MERGED_0@rory@expensify.com`, so this is just a utility specifically to strip that prefix.

Reply:
So to answer your question, yes the intention was to pass this only a single login at a time.

Reply:
Ok cool then this is fine. Just making sure we weren't looking for a global find and replace.